projects/1.18/assets/reliquary-v1-3/reliquary/lang/zh_cn.json

```
@@ -1 +1,362 @@
+    "jei.reliquary.description.rib_bone": "肋骨是用于合成的掉落物之一。\\n\\n由骷髅掉落。",
+    "jei.reliquary.description.withered_rib": "凋零肋骨是用于合成的掉落物之一。\\n\\n由凋灵骷髅掉落。",
+    "jei.reliquary.description.frozen_core": "霜冻核心是用于合成的掉落物之一。\\n\\n由雪傀儡掉落。",
+    "jei.reliquary.description.nebulous_heart": "朦胧之心是用于合成的掉落物之一。\\n\\n由末影人掉落。",
+    "jei.reliquary.description.squid_beak": "鱿鱼嘴是用于合成的掉落物之一。\\n\\n由鱿鱼掉落。",
+    "jei.reliquary.description.infernal_claw": "炼狱之爪是用于进一步配方的合成材料。",
+    "jei.reliquary.description.kraken_shell_fragment": "海妖壳甲残片是用于进一步配方的合成材料。",
+    "jei.reliquary.description.crimson_cloth": "腥红之布是用于进一步配方的合成材料。",
+    "jei.reliquary.description.guardian_spike": "守卫者之刺是用于合成的掉落物之一。\\n\\n由守卫者掉落。",
+    "jei.reliquary.description.chelicerae": "螯肢是用于合成的掉落物之一。\\n\\n由蜘蛛（包括普通和洞穴）掉落。",
+    "jei.reliquary.description.catalyzing_gland": "催化腺体是用于合成的掉落物之一。\\n\\n由苦力怕和恶魂掉落。",
+    "jei.reliquary.description.slime_pearl": "黏液珍珠是用于合成的掉落物之一。\\n\\n由史莱姆掉落。",
+    "jei.reliquary.description.bat_wing": "蝙蝠翅膀是用于合成的掉落物之一。\\n\\n由蝙蝠掉落。",
+    "jei.reliquary.description.zombie_heart": "僵尸之心是用于合成的掉落物之一。\\n\\n由僵尸（包括普通和猪灵）掉落。",
+    "jei.reliquary.description.molten_core": "熔火核心是用于合成的掉落物之一。\\n\\n由岩浆怪掉落。",
+    "jei.reliquary.description.eye_of_the_storm": "风暴之眼是用于合成的掉落物之一。\\n\\n由高压苦力怕掉落。",
+    "jei.reliquary.description.fertile_essence": "肥沃精华是用于进一步配方的合成材料。",
+    "jei.reliquary.description.phoenix_down": "放置在物品栏，死亡时满血复活，并获得一系列Buff。\\n起效后变为天使之羽，具有天使之羽的所有功能。",
+    "jei.reliquary.description.empty_potion_vial": "在药水酿造中使用空的药水瓶。",
+    "jei.reliquary.description.potion": "有许多不同种类的可饮用药水，查看药水精华介绍获取更多信息。",
+    "jei.reliquary.description.potion_essence": "药水精华有很多种类，它使用两种或三种成分（其中一种可以是另一类药水精华）的组合在药师研钵中制成。\\n\\n你在JEI看到的药水精华只是所有种类的药水精华的一小部分。它也只列出制作了一种效果的药水精华的配方，但你可以通过不同材料的配合制作拥有多种效果的药水精华。\\n\\n现在你可以在配置文件中看到它们可能的成分和负面效果。给予负面效果两个及以上的成分可以转化为正面效果。这样就能从研钵中得到药水精华了。",
+    "jei.reliquary.description.splash_potion": "有许多不同种类的飞溅药水，查看药水精华介绍获取更多信息。",
+    "jei.reliquary.description.lingering_potion": "浓缩的滞留药水，查看药水精华介绍和JEI配方获取更多药水制作信息。",
+    "jei.reliquary.description.pyromancer_staff": "烈焰术士之杖可以发射小火球（烈焰人）和火球（恶魂），用烈焰粉和火焰弹充能。\\n\\n拿着它可以完全防止恶魂或烈焰人发射的火球的伤害。此外，在它未激活时可以扑灭玩家周围的火。\\n\\n最后，你还可以将它作为打火石使用。",
+    "jei.reliquary.description.rending_gale": "按住右键，可以使玩家向着鼠标指向的方向飞行。请注意，它本身没有缓落效果。但如果你保持按住右键状态，鼠标指向地面，便可以取消摔落伤害。\\n\\n它同时还可以在玩家指向的方向推、拉实体和在雨天召唤闪电。使用Shift+滚轮切换模式。\\n\\n使用羽毛来为其充能。",
+    "jei.reliquary.description.rod_of_lyssa": "升级版钓鱼竿，可以将实体拉的更高，并且在潜行时，可以偷窃实体携带的物品。也能拉动掉在地上的物品。",
+    "jei.reliquary.description.salamander_eye": "当手持沙罗曼蛇之眼时，附近的火焰（和你自己身上的，如果适用）会被扑灭，恶魂的火球会被自动弹回，烈焰人的火球也会熄灭，但是烈焰人在你身旁时仍能伤害你。",
+    "jei.reliquary.description.serpent_staff": "向你的敌人喷出快速且粘稠的毒液，当被击中时候，生物将会被施加Debuff，同时渲染一个15秒的负面粒子效果（无害）。",
+    "jei.reliquary.description.shears_of_winter": "拥有范围剪切效果的剪刀。左键点击可以像普通剪刀一样使用。而右键使用时，则会对玩家周围进行剪切工作，该模式下，它可以剪羊毛，破坏树叶。",
+    "jei.reliquary.description.sojourner_staff": "可以存储多达1500根火把。\\n\\n可以使用它来远距离放置火把/方块，最远距离为30格，但是每距离6格，放置火把/方块的“代价”将会增加（因此，在30格外放置火把，将消耗6根火把）。\\n\\n如果同时持有虚妄提灯，提灯会尝试使用手杖中保存的火把。",
```

```suggestion
    "jei.reliquary.description.sojourner_staff": "可以存储多达1500根火把。\\n\\n可以使用它来远距离放置火把/方块，最远距离为30格，但是每距离6格，放置火把/方块的“代价”将会增加（因此，在30格外放置火把，将消耗6根火把）。\\n\\n如果同时持有诞妄提灯，提灯会尝试使用手杖中保存的火把。",
```
同一个提灯，三个位置的名字都不一样。上点儿心吧海燕呐